src/components/ReviewPage.jsx

```diff
@@ -62,24 +66,27 @@ function ReviewPage() {
                     <hr></hr>
                     <div className="review-page_reactions">
                         <h3>Did you like this review?</h3>
-                        <button className="review-page_reaction love" type="button" aria-label="Add 2 to the review impressions" onClick={() => {
-                            handleVote(2)
-                            }}>Love</button>
-                        <button className="reaction-page_reaction like" type="button" aria-label="Add 1 to the review impressions" onClick={() => {
-                            handleVote(1)
-                            }}>Like</button>
-                        <button className="reaction-page_reaction dislike" type="button" aria-label="Remove 1 from the review impressions" onClick={() => {
-                            handleVote(-1)
-                            }}>Dislike</button>
-                        <p>{voteCount} impressions</p>
+                        {error !== '' && showModal === true && <Modal text={error} setShowModal={setShowModal}/>}
+                        <button className="reaction-page_reaction like" type="button" aria-label="Add 1 to the review likes" onClick={() => {
+                            if (!reacted) {
+                                handleVote(1);
+                                setReacted(true);
+                                setShowModal(true);
+                            }}}></button>
+                        <button className="reaction-page_reaction dislike" type="button" aria-label="Remove 1 from the review likes" onClick={() => {
+                            if (!reacted) {
+                                handleVote(-1);
+                                setReacted(true);
+                                setShowModal(true);
+                            }}}></button>
+                        <p>{voteCount} likes</p>
+                        {reacted && showModal && <Modal text={"Whoops! Sorry, you can only react once"} setShowModal={setShowModal}/>}
                     </div>
                     <hr></hr>
```

Comment:
I really like the idea behind this, but it doesnt seem to work the way you may think.

Clicking the like or dislike buttons the first time shows an error modal pop up, which i dont think should be the case.

After clicking the like or dislike the opposite can be clicked but does not change the vote amount.

If you click one of the buttons and then click else where on the page the button that was selected is no longer obvious, but that is just something to consider at this point.